addons/calendar/models/calendar_attendee.py

```diff
@@ -68,7 +68,7 @@ def create(self, vals_list):
                 values['email'] = email[0] if email else ''
                 values['common_name'] = values.get("common_name")
         attendees = super().create(vals_list)
-        attendees._subscribe_partner()
+        attendees.sudo()._subscribe_partner()
```

Comment:
todo: try to do it differently.

Reply:
@arj-odoo if ``_subscribe_partners`` is considered safe, it may call ``_message_subscribe`` that is the private version of ``message_subscribe`` and that does not check ACLs (optimized version but does not care about rights as stated in docstring).